### DISCUSSION TITLE
New pre-defined destination: Clipboard

### DISCUSSION THREAD

**Discussion by jakobbg** [will-stone/browserosaurus#566] [Ideas]
Some apps just launch default browser with a URL. It would be great to catch this original URL before being redirected further on the web. So, instead of Safari (S) og Edge (E) be able to configure Clipboard (C).

Thank you for a great app! Simple, easy and beautiful.

**Comment by will-stone** [comment]
`cmd+c` or click the URL bar 👍

**Reply by jakobbg** [reply]
Sorry, what? I cannot see where to click in your app to facilitate url copying. I tried cmd+c while having app window open, but nothing was copied. And I see no url bar either.

**Reply by will-stone** [reply]
When the picker is open, press `cmd+c` or click the url at the bottom of the picker window. You should then get a notification that the url has been copied. Please provide a video of what you are trying if this is not working, thanks 😊

**Reply by mathieutu** [reply]
This is great! I was looking for that for a while!

Couldn't it be cool to document it somewhere, so people know?

**Comment by jakobbg** [comment]
It was working all along, but no was too long a link (from MS Teams) to paste into Spotlight afterwards, so nothing happened. So this is now confirmed to be working as intended. Perhaps add a small visual notification somehow to indicate the link actually has been copied? Now the window just disappears and no action is observed. Thank you for the guidance and kind help.

**Reply by will-stone** [reply]
<img width="348" alt="image" src="https://user-images.githubusercontent.com/654103/187960576-101718f9-d02f-4761-846b-c854c33cb515.png">

B sends a notification when copying a link.

**Reply by pressdarling** [reply]
@will-stone I wasn't aware of this functionality despite using the app for several months on all of my machines (I think I owe you a second coffee by now). Might I suggest some helper text on the dialog - "copy URL to clipboard", "⌘-C or click to copy URL", or similar?

<img width="246" alt="image" src="https://user-images.githubusercontent.com/32807920/190299254-9d357ed0-8677-4544-abd4-fcec16b8a739.png">

Happy to move this to an issue, not sure how simple it would be to add given the spacing but happy to help if possible.

**Reply by will-stone** [reply]
Hi @pressdarling Thank you for the support 🥳 I agree it's not obvious, but then I know it's there. And now that you do too, would you want it taking up precious UI real-estate? 🙂 Issues are for bugs so no need to open anything there. I could add a copy-to-clipboard option in the menu-bar menu, to increase discoverability, but it's not high priority... because I know it's there 😏 Happy to review a PR for that though, if you're interested?

**Reply by riotrah** [reply]
Would be willing to open a PR... At some point... Given my familiarity with the source, but in case someone else is also interested, I'd recommend a `pointer` cursor and "copy" icon somewhere when hovering over the link. 

Not as good as an ever-present indication of the feature but at least offers something in case one happens to have their mouse hover over the URL 🤷

**Reply by riotrah** [reply]
Can also confirm the code is very easy to contribute to with some react know-how

**Comment by jakobbg** [comment]
Evidently it helps to enable notifications for B. And with this, the app is
complete! ❤️

tor. 1. sep. 2022 kl. 18:03 skrev Will Stone ***@***.***>:
